cmd/util/integrityattestation/attestations.go

```diff
@@ -0,0 +1,212 @@
+package integrityattestation
```

Comment:
Give these files a clear folder name to tell users they are specific to Espresso?